### PR TITLE
#166790051: Social login 

### PR DESCRIPTION
**Pull request by kagabof**
#### What does this PR do?

- Enable login with social media.

#### Description of Task to be completed?

- Add redux action.
- Add redux reducer.
- Add component  

#### How should this be manually tested?

- By trying to access our application using social media login.
 

#### What are the relevant pivotal tracker stories?

[#166790051](https://www.pivotaltracker.com/story/show/166790051)

### CODE REVIEW

**Review by shaluchandwani — APPROVED** [review]

File: src/__tests__/components/Login.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { Login } from '../../containers/Login';
+
+const props = {
+  location: {},
+  socialLogin: jest.fn(),
+};
+
+describe('test logIn page', () => {
+  const wrapper = mount(
+  // <Provider store={store}/>
```
Comment (Joe-Joseph) [inline src/__tests__/components/Login.test.js]: Please remove the commented line

File: src/__tests__/components/index.test.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+it('Should pass', () => {
+  expect(5 * 5).toEqual(25);
+});
```
Comment (Joe-Joseph) [inline src/__tests__/components/index.test.js]: We no longer need this test

File: src/__tests__/components/login.test.js [hunk 3]
```diff
@@ -1,32 +1,21 @@
...
-});
-
-describe('<Error />', () => {
-  it('Should contain <span> to display error', () => {
-    const wrapper = shallow(<Error errors/>);
-    wrapper.setProps({ errors: { email: 'Email error' } });
-    expect(wrapper.contains(<span>{ errors }</span>)).to.equal(true);
+describe('test logIn page', () => {
+  const wrapper = mount(
+  // <Provider store={store}/>
```
Comment (Joe-Joseph) [inline src/__tests__/components/login.test.js]: same here remove the commented line

File: src/containers/signupContainer.js [hunk 4]
```diff
@@ -2,12 +2,20 @@
 /* eslint-disable react/prop-types */
```
Comment (Joe-Joseph) [inline src/containers/signupContainer.js]: If think you have defined props, you should enable this rule

File: src/redux/actions/actionTypes.js [hunk 5]
```diff
@@ -0,0 +1,2 @@
+export const SOCIAL_LOGIN = 'OCIAL_LOGIN';
```
Comment (billkabanga) [inline src/redux/actions/actionTypes.js]: Is the string supposed to be `OCIAL_LOGIN`?

**Review by mnzube — APPROVED** [review]

**Review by Joe-Joseph — APPROVED** [review]

**Pull request merged**